app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,34 @@
+window.onload = init();
+var id;
+
+function init(){
+  document.getElementById("upvote").addEventListener("click", upVote);
+  document.getElementById("downvote").addEventListener("click", downVote);
+  id = $("#api_id").val();
+}
+
+function upVote() {
+  var request = $.ajax({
+    method: "GET",
+    url: "/v1/upvote",
```

Comment:
The path here should be `api/v1`...check out how your namespacing is different from this example I gave you: https://github.com/mlg-/surrealist_poetry_api/blob/master/app/controllers/api/v1/poems_controller.rb